src/main/java/net/shadowmage/ancientwarfare/structure/item/ItemBlockInfo.java

```diff
@@ -2,16 +2,20 @@
 
 import net.minecraft.block.Block;
 import net.minecraft.block.state.IBlockState;
+import net.minecraft.client.util.ITooltipFlag;
 import net.minecraft.entity.player.EntityPlayer;
 import net.minecraft.item.ItemStack;
 import net.minecraft.util.ActionResult;
 import net.minecraft.util.EnumActionResult;
 import net.minecraft.util.EnumHand;
 import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.text.TextComponentString;
 import net.minecraft.world.World;
 import net.shadowmage.ancientwarfare.core.util.BlockTools;
 import net.shadowmage.ancientwarfare.core.util.WorldTools;
-import net.shadowmage.ancientwarfare.structure.AncientWarfareStructure;
+
+import javax.annotation.Nullable;
+import java.util.List;
 
 public class ItemBlockInfo extends ItemBaseStructure {

```

Comment:
I was actually going to remove this item in the future as I didn't see any purpose for it now that info tool does a lot of things. or is it used in some way?

Reply:
Should I just remove it then? What I saw as a difference, is that it dumps the tileentity info as well to the chat, not sure if that makes it worth to keep though